src/App/App.scss

```diff
@@ -1205,6 +1205,10 @@ button.App-button {
   align-items: center;
 }
 
+.chart-positions-settings {
```

Comment:
Is there a rule that determines where we place the css classes?

I mean, here ```.chart-positions-settings```
Defined in App.css, although we also have Exchange.css (and I think chart-positions refers to the Exchange page?)

Reply:
We don't at this point of time. I generally try to keep the css class as close as possible to the place it is being used.

In this case we have a modal on app which will contain the setting to have orders visible on the chart or not. So I think it is a part of App more than the exchange. The setting is related to the exchange but visible on the app.